gke-to-gke-vpn/terraform/main.tf

```diff
@@ -21,146 +21,153 @@ Creating Clusters and their Node Pools
 # Gets the current version of Kubernetes engine
 data "google_container_engine_versions" "gke_version" {
   location = "us-east1-b"
-  project  = "${var.project}"
+  project  = var.project
 }
 
 // Install the first cluster
 resource "google_container_cluster" "cluster-deployment-cluster1" {
   name               = "cluster-deployment-cluster1"
-  project            = "${var.project}"
-  location           = "${var.cluster1-location}"
-  network            = "${google_compute_network.network1.self_link}"
-  subnetwork         = "${google_compute_subnetwork.subnet1-us-east1.self_link}"
+  project            = var.project
+  location           = var.cluster1-location
+  network            = google_compute_network.network1.self_link
+  subnetwork         = google_compute_subnetwork.subnet1-us-east1.self_link
   initial_node_count = "1"
-  node_locations     = []
-  min_master_version = "${data.google_container_engine_versions.gke_version.latest_master_version}"
+  min_master_version = data.google_container_engine_versions.gke_version.latest_master_version
+
   ip_allocation_policy {
-    use_ip_aliases     = "true"
-    cluster_ipv4_cidr_block = "${var.cluster1-cidr}"
-    services_ipv4_cidr_block = "${var.cluster1-srv-cidr}"
+    use_ip_aliases           = "true"
+    cluster_ipv4_cidr_block  = var.cluster1-cidr
+    services_ipv4_cidr_block = var.cluster1-srv-cidr
   }
 }
 
 // Install node-pool for the first cluster. It's recommended by Terraform to be in a seperate block than main cluster
 resource "google_container_node_pool" "cluster1_nodes" {
-    name       = "cluster1-nodes"
-    location   = "${var.cluster1-location}"
-    project    = "${var.project}"
-    cluster    = "${google_container_cluster.cluster-deployment-cluster1.name}"
-    node_config {
-      oauth_scopes = [
-        "https://www.googleapis.com/auth/compute",
-        "https://www.googleapis.com/auth/devstorage.read_only",
-        "https://www.googleapis.com/auth/logging.write",
-        "https://www.googleapis.com/auth/monitoring",
-      ]
-      image_type   = "COS"
-      tags = ["kc-node"]
-    }
+  name     = "cluster1-nodes"
+  location = var.cluster1-location
+  project  = var.project
+  cluster  = google_container_cluster.cluster-deployment-cluster1.name
+
+  node_config {
+    oauth_scopes = [
+      "https://www.googleapis.com/auth/compute",
```

Comment:
We have to remove the `compute` scope wherever we see it.  Let's remove it here.

Reply:
```
    oauth_scopes = [
      "https://www.googleapis.com/auth/devstorage.read_only",
      "https://www.googleapis.com/auth/logging.write",
      "https://www.googleapis.com/auth/monitoring",
      "https://www.googleapis.com/auth/servicecontrol",
      "https://www.googleapis.com/auth/service.management.readonly",
      "https://www.googleapis.com/auth/trace.append",
    ]
```
is the newest/best defaults now.  Please apply to all 4.  Thanks!